go/common/crypto/signature/signers/ledger/ledger_signer.go

```diff
@@ -32,9 +33,13 @@ var (
 	// SignerDerivationRootPath is the derivation path prefix used for
 	// generating the signature key on the Ledger device.
 	SignerDerivationRootPath = []uint32{ledgerCommon.PathPurpose, SignerPathCoinType, SignerPathAccount, SignerPathChange}
+	// ConsensusDerivationRootPath is the derivation path prefix used for
+	// ledger Validator app.
+	ConsensusDerivationRootPath = []uint32{ledger.PathPurposeConsensus, SignerPathCoinType, SignerPathAccount, SignerPathChange}
```

Comment:
Should this point to `ledgerCommon.PathPurposeConsensus`? Also the existing `ledgerCommon.PathPurpose` should be renamed to something like `ledgerCommon.PathPurposeEntity` (to signify that it is for use with the entity key while the consensus one is used with the consensus key).

Reply:
Ok, fixing that!